python3_user_transition_guide.rst

```diff
@@ -0,0 +1,243 @@
+Python 3 transition guide
```

Comment:
I suggest linking to python-future.org

Also, your own "stop writing Python-4 incompatible code". =)